frontend/components/Slider/index.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Link from '@shopgate/pwa-common/components/Link';
+import Slider from '@shopgate/pwa-common/components/Slider';
+import styles from './style';
+
+/**
+ * Renders the component.
+ * @param {Object} props The component props.
+ * @returns {JSX}
+ */
+const SliderWidget = (props) => {
+  /**
+  * Creates an Item for a Slide.
+  * @param {Object} slideItem The Slide Item data.
+  * @return {JSX} The Slide Item.
+  */
+  const createSliderItem = (slideItem) => {
+    if (slideItem.img === '' || slideItem.img === undefined) {
+      return (
+        <img key="" className={styles.item} alt="" src={slideItem} />
+      );
+    }
+
+    if (slideItem.link === '' || slideItem.link === undefined) {
+      return (
+        <img key="" className={styles.item} alt="" src={slideItem.img} />
+      );
+    }
+
+    return (
+      <Link
+        href={slideItem.link}
+        key=""
+      >
+        <img key="" className={styles.item} alt="" src={slideItem.img} />
+      </Link>
+    );
+  };
+
+  /**
+  * Creates an Slide with Items.
+  * @param {Object} slide The Slide data.
+  * @return {JSX} The rendered Slide.
+  */
+  const createSlide = slide => (
+    <div key="">
+      <Slider.Item>
+        <div className={styles.sliderWrapper}>
+          {slide.map((
+            slideItem => createSliderItem(slideItem)
+          ))}
+        </div>
+      </Slider.Item>
+    </div>
+  );
+
+  const slides = props.settings.slides.map((
+    slide => createSlide(slide)
+  ));
+
+  return (
```

Comment:
if props.settings.slides is empty it should render nothing here

Reply:
done